logging/logmanager/plugin_impl_log_manager.go

```diff
@@ -87,7 +96,6 @@ func (lm *Plugin) Init() error {
 		if err != nil {
 			return err
 		}
-		lm.Log.Debugf("logs config: %+v", lm.Conf)
```

Comment:
Removing this why?

Reply:
Ups, deleted by mistake with my temp debugs.

Reply:
It's still removed..